code/datums/chat_message.dm

```diff
@@ -239,17 +239,17 @@
 	m *= 255
 	switch(h_int)
 		if(0)
-			return "[rgb(c,x,m)]C8"
+			return "[rgb(c,x,m)]"
```

Comment:
No need to embed this in a string anymore, since `rgb()` already returns a string

Reply:
Done